### PR TITLE
fix: Resolve application crash when viewing Asset Modification Request

### LINKED ISSUE #389: [Feature] Mark the mandatory fields in "Create Asset" Drawer
**Is your feature request related to a problem? Please describe.**
The user may not know which fields are mandatory and which are not.

**Describe the solution you'd like**
Mark the mandatory fields with a '*'.

**Additional context**
- Mark the mandatory fields in "Modify Asset" Drawer as well

### LINKED ISSUE #393: [Bug] Application crash on approval of asset modification
**Describe the bug**
When opening the request for Asset Modification, the entire application crashes.

**To Reproduce**
1. Login as System Admin
2. Modify an existing asset
3. Login as Lead
4. Go to "Approve Assets" -> "In Modification"
5. Click on the View icon of an asset

**Expected behavior**
When the view icon of the asset is clicked, it should show the details of the asset along with highlighting the modified fields.

**Additional Context**
- This may be due to the logs not being created when the assets are imported.

### PR DESCRIPTION
**Pull request by AnanthanCJ**
Fixes #393 
Fixes #389

### CODE REVIEW

File: exam_frontend/src/components/CardComponent/CardComponent.tsx [hunk 1]
```diff
@@ -467,8 +467,8 @@ const CardComponent: React.FC<CardType> = ({
     {
       label: "Asset Category",
       value: (
-        <Form.Item name="assetCategory">
-          <b style={{ display: "block" }}>Asset Category: </b> <br></br>
+        <Form.Item name="assetCategory" rules={[{ required: true, message: 'Product Name is required' }]}>
```
Comment (getwithashish) [inline exam_frontend/src/components/CardComponent/CardComponent.tsx]: Could you correct this to "Asset Category"?

File: exam_frontend/src/components/CardComponent/CardComponent.tsx [hunk 2]
```diff
@@ -559,8 +559,8 @@ const CardComponent: React.FC<CardType> = ({
       label: "Serial Number",
       name: "serialNumber",
       value: (
-        <Form.Item name="serial number">
-          <b>Serial Number:</b> <br></br>
+        <Form.Item name="serial number" rules={[{ required: true, message: 'Product Name is required' }]}>
+          <b>Serial Number: <span style={{ color: 'red' }}>*</span></b> <br></br>
```
Comment (getwithashish) [inline exam_frontend/src/components/CardComponent/CardComponent.tsx]: Could you correct this to Serial Number?

**Review by getwithashish — CHANGES_REQUESTED** [review]
Please have a look at the suggestions.

File: exam_frontend/src/components/CardComponent/CardComponent.tsx (hunk 1 above)
Comment (AnanthanCJ) [reply exam_frontend/src/components/CardComponent/CardComponent.tsx]: sure

File: exam_frontend/src/components/CardComponent/CardComponent.tsx (hunk 2 above)
Comment (AnanthanCJ) [reply exam_frontend/src/components/CardComponent/CardComponent.tsx]: sure

**Review by getwithashish — APPROVED** [review]
LGTM

File: exam_frontend/src/components/CardComponent/CardComponent.tsx (hunk 1 above)
Comment (getwithashish) [reply exam_frontend/src/components/CardComponent/CardComponent.tsx]: Thank you

File: exam_frontend/src/components/CardComponent/CardComponent.tsx (hunk 2 above)
Comment (getwithashish) [reply exam_frontend/src/components/CardComponent/CardComponent.tsx]: Thank you

**Pull request merged**